src/plugins/intel_gna/src/transformations/insert_copy_layer.cpp

```diff
@@ -199,7 +199,8 @@ InsertCopyBeforeConcatLayer::InsertCopyBeforeConcatLayer() {
             // Crop -> Concat, Input -> Split -> Concat
             if ((std::dynamic_pointer_cast<ngraph::op::CropIE>(current_node) && !is_crop_affined(current_node)) ||
                 std::dynamic_pointer_cast<ngraph::opset8::Split>(current_node) ||
-                std::dynamic_pointer_cast<ngraph::opset8::VariadicSplit>(current_node)) {
+                std::dynamic_pointer_cast<ngraph::opset8::VariadicSplit>(current_node) ||
+                std::dynamic_pointer_cast<ngraph::opset9::FakeQuantize>(current_node)) {
```

Comment:
If we change the transformation patterns it is required to update description in the header. 
Please update the header too.